src/CsvHelper/CsvWriter.cs

```diff
@@ -37,7 +37,8 @@ public class CsvWriter : ICsvWriter
 		private readonly Dictionary<Type, TypeConverterOptions> typeConverterOptionsCache = new Dictionary<Type, TypeConverterOptions>();
 		private readonly ICsvWriterConfiguration configuration;
 		private int row = 1;
-		private CsvPropertyMapData reusablePropertyMapData = new CsvPropertyMapData( null );
+		private readonly CsvPropertyMapData reusablePropertyMapData = new CsvPropertyMapData( null );

```

Comment:
Seems like this line was picked up by the diffing algorithm when it shouldn't have been. just needs to not do this